validphys2/src/validphys/tests/test_pseudodata.py

```diff
@@ -38,8 +38,17 @@ def test_read_pdf_pseudodata():
 
 
 def test_recreate_fit_pseudodata():
-    fit_pseudodata = API.recreate_fit_pseudodata(fit=PSEUDODATA_FIT)
-
+    config = {}
+    config["use_t0_sampling"] = False
+    config["separate_multiplicative"]= True
+    config["output_path"] = None
+    config["theory_covmat_flag"] = False
+    config["use_user_uncertainties"] = None 
+    config["use_scalevar_uncertainties"] = None
+    config["use_thcovmat_in_sampling"]=None
+    config["dataset_t0_predictions"] = None
```

Comment:
Why do you need this? `recreate_fit_pseudodata` should be ale to recreate it from the fit only.